app/util/env.ts

```diff
@@ -65,58 +42,66 @@ const envVars: HarmonyEnv = {} as HarmonyEnv;
  * @param defaultValue - The value to use if the environment variable is not set. Only strings
  *   and integers are supported
  */
-function makeConfigVar(envName: string, defaultValue?: string | number): void {
-  const envValue = process.env[envName];
-  let value;
-
-  if (!envValue) {
-    value = defaultValue;
-  } else if (typeof defaultValue === 'number') {
-    value = parseInt(envValue, 10);
+function makeConfigVar(envName: string, defaultValue?: string): void {
+  const stringValue = process.env[envName] || defaultValue;
+  if (isInteger(stringValue)) {
+    envVars[_.camelCase(envName)] = parseInt(stringValue, 10);
   } else {
-    value = envValue;
+    envVars[_.camelCase(envName)] = stringValue;
   }
+  process.env[envName] = stringValue;
+}
 
-  envVars[_.camelCase(envName)] = value;
+const allEnv = { ...envDefaults, ...envOverrides, ...process.env };
+
+for (const k in allEnv) {
```

Comment:
Does it not work to do:

```javascript
for (const k of Object.keys(allEnv)) {
```

And then remove the linter comment at the top of the file?  `for ... in` can be problematic in surprising ways, though this usage should be safe